### PR TITLE
Make logging for gcl e2e test more verbose

### PR DESCRIPTION
**Pull request by crassirostris**
To help debug https://github.com/kubernetes/kubernetes/issues/37241

CC @piosz

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37634)

### CODE REVIEW

File: test/e2e/cluster_logging_gcl.go [hunk 1]
```diff
@@ -31,12 +31,20 @@ import (
...
+)
+
 var _ = framework.KubeDescribe("Cluster level logging using GCL", func() {
 	f := framework.NewDefaultFramework("gcl-logging")
 
 	BeforeEach(func() {
 		// TODO (crassirostris): Expand to GKE once the test is stable
 		framework.SkipUnlessProviderIs("gce")
+
+		increaseFluentdVerbosity(f)
```
Comment (piosz) [inline test/e2e/cluster_logging_gcl.go]: Please add todo to remove it.

**Comment by piosz** [comment]
LGTM, just a nit

File: test/e2e/cluster_logging_gcl.go (hunk 1 above)
Comment (Crassirostris) [inline test/e2e/cluster_logging_gcl.go]: Done

**Comment by Crassirostris** [comment]
@k8s-bot node e2e tests this

**Comment by Crassirostris** [comment]
@k8s-bot test this

**Comment by rmmh** [comment]
@k8s-bot verify test this

**Pull request merged**